tests/fuzzer/CMakeLists.txt

```diff
@@ -0,0 +1,112 @@
+cmake_minimum_required(VERSION 3.9)
```

Comment:
If this entirely subsumes the shell scripts in this dir, should we remove those?